ip_ranges.go

```diff
@@ -0,0 +1,45 @@
+package netaddr
+
+import (
+	"fmt"
+	"net"
+)
+
+// IPRange range of ips not necessarily aligned to a power of 2
+type IPRange struct {
+	First, Last net.IP
+}
+
+func (r *IPRange) String() string {
+	return fmt.Sprintf("[%s,%s]", r.First, r.Last)
+}
+
+// IPRangeFromCIDR get an IP Range from a CIDR
+func IPRangeFromCIDR(cidr *net.IPNet) *IPRange {
+	return &IPRange{
+		First: cidr.IP,
```

Comment:
This got me thinking: #15 

I kind of think we should fix it and use it here. What do you think?

Reply:
maybe we should add a helper `normalizeNet` that sets the net.IP to be the network addr